modules/gdscript/gdscript.h

```diff
@@ -71,8 +71,8 @@ class GDScript : public Script {
 	friend class GDScriptFunction;
 	friend class GDScriptAnalyzer;
 	friend class GDScriptCompiler;
-	friend class GDScriptFunctions;
 	friend class GDScriptLanguage;
+	friend struct GDScriptUtilityFunctionsDefinitions;
```

Comment:
Is this necessary if the struct has no private members?

Reply:
This is so the struct can access the private members of the GDScript class (which is needed by some of the functions).